File: beecrowd/1008/1008 - Salary .cs
using System;

class Program
{
	static void Main(string[] args)
	{
		int nEmployee, hWorked;
		float hSalary;
		
		nEmployee = int.Parse(Console.ReadLine());
		hWorked = int.Parse(Console.ReadLine());
		hSalary = float.Parse(Console.ReadLine());
		
		float tSalary = hWorked * hSalary;
		
		Console.WriteLine("NUMBER = " + nEmployee);
		Console.WriteLine($"SALARY = U$ {tSalary:F2}");
	}
}

File: beecrowd/1008/1008 - Salary .cpp
#include <iostream>
#include <iomanip>

int main()
{
	int nEmployee, hWorked;
	float hSalary;

	std::cin >> nEmployee;
	std::cin >> hWorked;
	std::cin >> hSalary;

	float tSalary = hWorked * hSalary;

	std::cout << "NUMBER = " << nEmployee << std::endl;
	std::cout << std::fixed << std::setprecision(2) << "SALARY = U$ " << tSalary << std::endl;

	return 0;
}